.github/workflows/haskell.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       fail-fast: false
       matrix:
         ghc: ["8.10.7", "9.2.4"]
+        cabal: ["3.6.2.0"]
```

Comment:
Is there a reason we should be running our CI with different versions of cabal? I think this setting should go directly into `haskell/actions`

Reply:
Done!